sparkmagic/sparkmagic/livyclientlib/command.py

```diff
@@ -42,6 +42,19 @@ def execute(self, session):
             response = session.http_client.post_statement(session.id, data)
             statement_id = response[u'id']
             output = self._get_statement_output(session, statement_id)
+        except KeyboardInterrupt as e:
+            msg =  u'Interrupted by user'
```

Comment:
Still need to find a good place to print this message.

Reply:
What are the options you've considered?

Reply:
I only tried returning it, but remaining cells will continue to execute, which I also want to avoid. If I raised an exception, the backtrace is too long and not not helpful at all. Any suggestion?